.github/ISSUE_TEMPLATE/spike-template.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Spike Template
+about: Template for issues focused on research (spikes)
+title: 'SPIKE - '
+labels: Spike
+assignees: ''
+
+---
+
+## Description
+Short description of the issue you would like to solve. 
+
+## Requirements
+- Requirement 1
+- Requirement 2
+
+### Nice to have or Goal Requirements
+- Nice to have 1
+- Nice to have 2
+
+## Additional notes
+Any additional notes on the issue, if you have them. 
+
+## Related tickets
+- #123 
```

Comment:
Do you know if this will always automatically reference issue "123" once we get that many? I'm not sure the best way to go about not linking something but still giving an example of what you want, maybe just removing the "#"?
```suggestion
- 123 
```

Reply:
Yeah, good point. I'll change it.